exporter/exporter.go

```diff
@@ -169,9 +169,11 @@ func Run(args ...string) error {
 	}
 
 	// default to public API environment
-	lightstepEnv := "public"
-	if len(os.Getenv("LIGHTSTEP_ENV")) > 0 {
-		lightstepEnv = os.Getenv("LIGHTSTEP_ENV")
+	lightstepUrl := "https://api.lightstep.com"
+	if len(os.Getenv("LIGHTSTEP_API_URL")) > 0 {
+		lightstepUrl = os.Getenv("LIGHTSTEP_API_URL")
+	} else if len(os.Getenv("LIGHTSTEP_API_BASE_URL")) > 0 {
+		lightstepUrl = os.Getenv("LIGHTSTEP_API_BASE_URL")
 	}
```

Comment:
I just reviewed the tests and this appears to be untested now that we've moved it out of the client. Might need to hoist some of this environment handling out into a helper that you can unit test—unless there's a clear path to verify the configuration combinatorics in this `Run` function directly.

Reply:
Yeah, I agree testing `Run` is out of scope. I could make a case that since this aspect was somewhat under test before that we've lost some coverage, but it's relatively constrained and we can address in another pass if you'd like.

Reply:
Maybe i was just being lazy, I added a test for Run